CHANGELOG.rst

```diff
@@ -11,6 +11,28 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
 
+[2.0.0] - 2020-11-05
+~~~~~~~~~~~~~~~~~~~~
+
+* Deprecate namespaced waffle classes: the WaffleFlag and WaffleSwitch classes exposed in `toggles` are now the namespace-less classes which were previously in `toggles.__future__`.
+    * If you were importing from `edx_toggles.toggles.__future__` before, then you simply need to import from `edx_toggles.toggles`. Importing from `__future__` will continue to work but will trigger a deprecation warning.
+    * If you were importing from `edx_toggles.toggles`, then you either need to:
+        * Migrate your legacy namespaced classes to the new-style classes (see the new behaviour below).
+        * Import `LegacyWaffleFlag` instead of `WaffleFlag` and `LegacyWaffleSwitch` instead of `WaffleSwitch`.
+    * The new classes introduce the following changes:
+        * The `WaffleSwitchNamespace._namespaced_name` and `WaffleFlagNamespace._namespaced_name` methods are replaced by the `WaffleSwitch.name` and `WaffleFlag.name` attributes.
+        * The `WaffleSwitchNamespace.is_enabled` method is replaced by the `WaffleSwitch.is_enabled` method.
+        * The `WaffleSwitchNamespace.set_request_cache_with_short_name` method is deprecated; instead, you could (but really shouldn't) use the `WaffleSwitch._cached_switches` property.
+        * The `WaffleSwitch.switch_name` attribute is deprecated: switches should only ever be referred to using their fully namespaced names.
```

Comment:
I think the term "deprecated" is a little confusing from the standpoint of the new class.  How about?
```
    * The `WaffleSwitch.switch_name` attribute no longer exists.  Switches should only ever be referred to using their fully namespaced names.  If you need the non-namespaced name, it must be parsed from the namespaced name.
```